mycroft/skills/twitter/vocab/en-us/PostText.voc

```diff
@@ -0,0 +1,2 @@
+I am ready to start my day
+Mycroft is tweeting for me!
```

Comment:
Where is this vocabulary used for? Is that supposed to be a feedback (dialog) instead?